Content.Server/Atmos/Piping/Binary/EntitySystems/GasPassiveGateSystem.cs

```diff
@@ -85,5 +86,23 @@ private void OnExamined(EntityUid uid, GasPassiveGateComponent gate, ExaminedEve
             var str = Loc.GetString("gas-passive-gate-examined", ("flowRate", $"{gate.FlowRate:0.#}"));
             args.PushMarkup(str);
         }
+
+        /// <summary>
+        /// Returns the gas mixture for the gas analyzer
+        /// </summary>
+        private void OnAnalyzed(EntityUid uid, GasPassiveGateComponent component, GasAnalyzerScanEvent args)
+        {
+            if (!EntityManager.TryGetComponent(uid, out NodeContainerComponent? nodeContainer))
+                return;
+
+            var gasMixDict = new Dictionary<string, GasMixture?>();
+
+            if(nodeContainer.TryGetNode(component.InletName, out PipeNode? inlet))
+                gasMixDict.Add(component.InletName, inlet.Air);
+            if(nodeContainer.TryGetNode(component.OutletName, out PipeNode? outlet))
+                gasMixDict.Add(component.OutletName, outlet.Air);
+
+            args.GasMixtures = gasMixDict;
+        }
```

Comment:
Yeah this was all based on a prior discussion suggesting I made it event based instead of fetching component references. The 'default' is the fall-back for pipes since I couldn't figure out how to get them to react properly without resorting to component references.